documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc

```diff
@@ -117,9 +117,29 @@ You can add source providers and destination providers for a virtual machine mig
 [id="adding-source-providers"]
 ==== Adding source providers
 
-You can add a VMware source provider, {a-rhv} source provider, or an {osp} source provider by using the {ocp} web console.
+You can use {project-short} to migrate VMs from the following source providers:
+
+* {virt}
+* {osp}
+* {rhv-full}
+* VMware vSphere
```

Comment:
like the comment on the dialog, I think the order here should also highlight the importancy / relevancy:
```
* VMware vSphere
* {rhv-full}
* {osp}
* Open Virtual Appliances (OVAs)
* {virt}
```